go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/common-nighthawk/go-figure v0.0.0-20210622060536-734e95fb86be
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.8.1
 	github.com/moby/term v0.5.0
+	github.com/nozzle/throttler v0.0.0-20180817012639-2ea982251481
```

Comment:
The package seems to be unmaintained, do we have an alternative?

Reply:
Yeah, I also noticed that. I added it because it is super simple (less than 200 lines of code), has zero external deps and we have it on other releng projects. Happy to switch to something else, any suggestions?

Another option would be forking it to this repo and maintaining it ourselves since we're using it.

Reply:
We could move over the code if the license allows and maintain it as part of this repo.

Reply:
+1, either to move the code or to somehow fork the repository if moving the code is not possible

Reply:
It's released under apache2, I'll import the commits and add a credit line and we can adopt it from now on :")

Reply:
OK, I opened https://github.com/kubernetes-sigs/release-utils/pull/108. I'll update the imports in this one once we finish merging the thottler fork.